cli/src/pcluster/models/imagebuilder.py

```diff
@@ -332,12 +354,9 @@ def _upload_config(self):
         """Upload source config to S3 bucket."""
         try:
             if self.config:
-                # Upload config with default values
-                self.bucket.upload_config(config=self.config, config_name=self._s3_artifacts_dict.get("config_name"))
-
                 # Upload original config
                 self.bucket.upload_config(
-                    config=self.config.source_config, config_name=self._s3_artifacts_dict.get("source_config_name")
+                    config=self.__source_config, config_name=self._s3_artifacts_dict.get("config_name")
```

Comment:
You shouldn't access private attributes, it was the reason why we had source_config property.

Reply:
done